src/python-frontend/python_frontend_types.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+enum class StatementType
+{
+  VARIABLE_ASSIGN,
+  COMPOUND_ASSIGN,
+  FUNC_DEFINITION,
+  IF_STATEMENT,
+  WHILE_STATEMENT,
+  UNKNOWN,
+};
+
+enum class ExpressionType
+{
+  UNARY_OPERATION,
+  BINARY_OPERATION,
+  LITERAL,
+  VARIABLE_REF,
+  UNKNOWN,
+};
+
+static const std::unordered_map<std::string, std::string> operator_map = {
```

Comment:
Type definitions like the enums above are great to have in header files, but symbol definitions like this are bad in the sense that they get duplicated in every C++ translation unit this header is included in. Meaning that the resulting ESBMC binary would multiple versions of this symbol. In case this symbol is indeed used by multiple different C++ files, would you consider making this symbol `extern` and defining it only once in some translation unit? If it's used only once, please remove it from the header. Same applies to `statement_map` below.